src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -22,6 +22,7 @@ CA2012 | Reliability | Hidden | UseValueTasksCorrectlyAnalyzer, [Documentation](
 CA2013 | Reliability | Warning | DoNotUseReferenceEqualsWithValueTypesAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2013)
 CA2014 | Reliability | Warning | DoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2014)
 CA2015 | Reliability | Warning | DoNotDefineFinalizersForTypesDerivedFromMemoryManager, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2015)
+CA2016 | Reliability | Warning | ForwardCancellationTokenToAsyncMethodsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2016)
```

Comment:
Do we want this to be a build breaking warning (on /warnaserror)? I like the rule as such, but want to make sure that this was an explicit call made to flag these as warnings. Tagging @dotnet/dotnet-analyzers

Reply:
I forwarded this question to my team, waiting for an answer.